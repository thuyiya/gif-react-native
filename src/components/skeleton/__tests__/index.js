/* eslint-disable no-undef */
import React from "react";
// import TestRenderer from "react-test-renderer";
import ShallowRenderer from "react-test-renderer/shallow";
import { Skeleton } from "_components";

describe("Skeleton", () => {
  it("renders correctly react-test-renderer", () => {
    const renderer = new ShallowRenderer();
    renderer.render(<Skeleton />);
    const result = renderer.getRenderOutput();

    expect(result).toMatchSnapshot();
  });
});
