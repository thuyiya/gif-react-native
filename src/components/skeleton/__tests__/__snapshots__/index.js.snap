// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skeleton renders correctly react-test-renderer 1`] = `
<ForwardRef(AnimatedComponentWrapper)
  style={
    Object {
      "backgroundColor": "#f0f0f0",
      "height": "100%",
      "marginTop": 0,
      "opacity": 0.5,
      "width": "100%",
    }
  }
/>
`;
